binderhub/static/js/index.js

```diff
@@ -170,21 +177,21 @@ function indexMain() {
     return false;
   });
 
-  $("#build-form").submit(function () {
+  $("#build-form").submit(async function (e) {
+    e.preventDefault();
```

Comment:
This was necessary as `return false` doesn't work from async functions. Same effect

Reply:
I figure no - that the event to submit this form isnt considered by parent elements, but I'm not confident reviewing this in general.

Reply:
I don't think it's relevant.